src/Renderer/OBB.js

```diff
@@ -141,7 +141,7 @@ class OBB extends THREE.Object3D {
             };
 
             const geometry = new TileGeometry(paramsGeometry);
-            this.copy(builder.OBB(geometry.boundingBox));
+            this.copy(new OBB(geometry.boundingBox.min, geometry.boundingBox.max));
```

Comment:
I just noticed It's possible to remove the `OBB` instance:

```js
let obb;

//...
// this.copy(new OBB(geometry.boundingBox.min, geometry.boundingBox.max));
obb.box3D.copy(geometry.boundingBox);
obb.natBox.copy(obb.box3D);
this.copy(obb);
//...

obb = new OBB();
```

Reply:
I added a `copyBox3`, it is even simpler imho. See my latest commit.

Reply:
you need to `this.copy(obb);` because the transformation is copied too.

Reply:
I disagree, because at this point, `obb` should have the default value of `z`, so I don't understand how that is different from what I added ?

Reply:
In fact no transformation at all has been done at this point, so there is nothing to copy except the box3 ?

Reply:
if this obb instance  have transformation, so it needs to copy identity transformation.